CONTRIBUTING.md

```diff
@@ -115,7 +115,7 @@ If you'd like to test latest Kaoto UI and not rely on a released version, follow
   * Open the file
 
 To return to the default Kaoto UI version, just write on `vscode-kaoto` local clone folder:
-* `yarn unlink` _\<kaoto-ui local clone folder uri>_
+* `yarn unlink` _\<kaoto-ui local clone folder uri>/packages/kaoto-ui_
   * i.e. `yarn unlink ~/repositories/kaoto-ui`
```

Comment:
how should example on this line looks like? `yarn unlink ~/repositories/kaoto-ui/packages/kaoto-ui_` is this correct?

Reply:
updated :+1: